misp_modules/lib/vt_graph_parser/importers/base.py

```diff
@@ -0,0 +1,98 @@
+"""vt_graph_parser.importers.base.
+
+This module provides a common method to import graph from misp attributes.
+"""
+
+
+import vt_graph_api
```

Comment:
This one needs to be added in the pipenv config files and eventually in the requirements file, but no big deal we can do that after the PR is merged

Reply:
I will add it to those files.